src/app/features/add-new-post/add-new-post.component.ts

```
@@ -0,0 +1,301 @@
+      this.commissionValue = price * this.commissionRate;
+      this.netPrice = price - this.commissionValue;
+    } else {
+      this.commissionValue = 0;
+      this.netPrice = 0;
+    }
+  }
+
+  onDeleteImage(index: number): void {
+    // Remove from upload states
+    const uploadState = this.imageUploadStates[index];
+    if (uploadState && uploadState.imageId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)